packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Resolved bug that resulted in user not being prompted for a keyPassphrase if in the secure array of ssh profile. [#1770](https://github.com/zowe/zowe-cli/issues/1770)
+- Enhancement: SshBaseHandler will now prompt user up to 3 times to enter the correct keyPassphrase in the case that the stored value is incorrect or no value is stored. [#1770](https://github.com/zowe/zowe-cli/issues/1770)
```

Comment:
Can we explain what SShBaseHandler is? Is it a property, a class, a manufacturer?: "Enhancement: The **property/schema/cat/whatever** now prompts the user up to three times to ..."

Reply:
Do we need this enhancement in the Imperative changelog? Since it's specific to `SshBaseHandler`, I think it might make sense to include only in the z/OS USS changelog.